ephios/api/templates/api/access_token_list.html

```diff
@@ -4,95 +4,125 @@
+                        <div class="col-12 col-lg">
                             <h6>{% translate "Scope" %}</h6>
                             <ul>
-                                {% for scope_name, scope_description in authorized_token.scopes.items %}
+                                {% for scope_name, scope_description in token.scopes.items %}
                                     <li><code>{{ scope_name }}</code>: {{ scope_description }}</li>
                                 {% endfor %}
                             </ul>
                         </div>
+                        <div class="col-12 col-lg-auto">
+                            {% if token.revoked %}
+                                <span class="badge bg-danger text-white">{% translate "Revoked" %}</span>
+                            {% else %}
+                                <form method="POST" action="{% url "api:settings-access-token-revoke" %}">
+                                    {% csrf_token %}
+                                    <input type="hidden" name="pk" value="{{ token.id }}">
+                                    <button type="submit"
+                                            class="btn btn-outline-danger">{% translate "Revoke" %}</button>
+                                </form>
+                            {% endif %}
+                        </div>
                     </div>
                 </div>
             </div>
         {% empty %}
-            {% if request.GET.show_inactive %}
-                <p>{% translate "There are no tokens." %}</p>
-            {% else %}
-                <p>{% translate "There are no active tokens." %}</p>
-            {% endif %}
+            <p>{% translate "No applications have been authorized." %}</p>
         {% endfor %}
     </div>
+
+    <h2>
+        {% translate "Personal API tokens" %}
+    </h2>
+
+    <a href="{% url "api:settings-access-token-create" %}"
+       class="btn btn-secondary mb-3">{% translate "Create new API token" %}</a>
+
+    {% for token in personal_access_tokens %}
+        <div class="card w-100 mb-3">
+            <div class="card-body">
+                <div class="d-flex justify-content-between align-items-start">
+
+                </div>
+                <div class="row">
+                    <div class="col-12 col-md-4">
+                        <h5 class="card-title">
+                            {{ token.description|rich_text }}
+                        </h5>
+                        <p class="text-muted">
+                            {% blocktranslate trimmed with created=token.created %}
+                                created on {{ created }}
+                            {% endblocktranslate %}
+                            <br/>
+                            {% if token.revoked %}
+                                {% blocktranslate trimmed with revoked=token.expires %}
+                                    revoked on {{ revoked }}
```

Comment:
this isn't a date anymore, is it?

Reply:
its not, but the variable revoked in the translation is set to the expiration date used as surrugate